packages/backend/src/tokens/interstitial.ts

```diff
@@ -19,7 +19,10 @@ export type LoadInterstitialOptions = {
   domain?: string;
 };
 
-export function loadInterstitialFromLocal(options: Omit<LoadInterstitialOptions, 'apiUrl' | 'isSatellite' | 'domain' | 'proxyUrl'>) {
+export function loadInterstitialFromLocal(
+  options: Omit<LoadInterstitialOptions, 'apiUrl' | 'isSatellite' | 'domain' | 'proxyUrl'>,
+) {
+  options.frontendApi = parsePublishableKey(options.publishableKey)?.frontendApi || options.frontendApi || '';
```

Comment:
🎉 This fixes an issue i have with interstitial in remix app using publishableKey.